packages/graph/hash_graph/lib/graph/src/knowledge/link/mod.rs

```diff
@@ -64,29 +64,52 @@ impl Link {
 /// metadata.
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, ToSchema)]
 #[serde(rename_all = "camelCase")]
-pub struct PersistedLink {
-    inner: Link,
+pub struct PersistedLinkMetadata {
```

Comment:
Comment:
Since we don't have a dedicated update operation for links, we are currently losing information about the link owner when we update a link on the backend. I.e. a link is removed/added, effectively resetting the owner. 

Perhaps it would be relevant to flag in https://app.asana.com/0/1200211978612931/1203031430417465/f

Reply:
I think information loss also only applies once [this follow-up PR](https://github.com/hashintel/hash/pull/1197) is complete, but it's a good point which I've flagged in the task.